pkg/flags/flags.go

```diff
@@ -295,8 +295,19 @@ func (m LabelsMap) Set(val string) error {
 		if len(kv) != 2 {
 			return fmt.Errorf("label format error: %s", label)
 		}
-		m[strings.TrimSpace(kv[0])] = strings.TrimSpace(kv[1])
+		k := strings.TrimSpace(kv[0])
```

Comment:
Probably would be a good idea to save both key and values all lowercases.

Reply:
Yup, good idea!

Reply:
@embano1 Maybe just the key should be lowered and be case insensitive so that it makes searching for a label simpler (in the `Contains` method) or as an alternative use `strings.EqualFold` when comparing in `Contains`.

Reply:
Made `key` case-insensitive in `Set()` and `Contains()` and updated `flag` doc string.

Reply:
Reverted as per the PR discussion